NEWS.md

```diff
@@ -1,3 +1,8 @@
+### Tarsnap 1.0.40 (unreleased)
+
+- Fix `clock_gettime()` problem on OSX.
```

Comment:
This seems excessively vague.  Can we say something about OS X versions and XCode (which I think was implicated somehow)?